src/fastfield/serializer/gcd.rs

```diff
@@ -0,0 +1,176 @@
+use fastdivide::DividerU64;
+use gcd::Gcd;
+
+pub const GCD_DEFAULT: u64 = 1;
+
+pub fn find_gcd(mut numbers: impl Iterator<Item = u64>, base: u64) -> Option<u64> {
```

Comment:
Can we remove base from here and put that in the caller?

Reply:
yep I changed it